addons/web_editor/static/src/js/editor/odoo-editor/test/editor-test.js

```diff
@@ -31,36 +32,10 @@ mocha.run(failures => {
         console.log('test successful');
     }
 
-    // Clean report of successful tests to ease DEV / debug
-    // just add ?clean=1
-    const reportEl = document.getElementById('mocha-report');
-    window.scrollTo(0, window.scrollY + reportEl.getBoundingClientRect().top);
-    const urlParams = new URLSearchParams(window.location.search);
-    if (urlParams.get('clean')) {
-        console.log('Successful tests have been removed.');
-        const report = document.querySelector("#mocha-report");
-
-        let hr = document.createElement("hr");
-        report.prepend(hr);
-        let h1 = document.createElement("h1");
-        h1.textContent = "Clean : Successfull tests have been removed.";
-        report.prepend(h1);
-
-        const passedTests = report.querySelectorAll('.test.pass');
-        for (let el of passedTests) {
-            el.remove();
-        }
-
-        const cleanUl = function () {
-            const emptyUl = report.querySelectorAll('ul:empty');
-            for (let ul of emptyUl) {
-                ul.parentElement.remove();
-            }
-        };
-        for (let i = 4; i > 0; i--) {
-            cleanUl();
-        }
-
-        report.outerHTML = report.outerHTML.replaceAll('//zws//', '<b class="zws">zws</b>');
-    }
+    // Better visualisation of invisible character in test report.
+    const report = document.querySelector("#mocha-report");
+    let reportHtml = report.outerHTML
+    reportHtml = reportHtml.replaceAll('//zws//', '<b class="zws">zws</b>');
+    reportHtml = reportHtml.replaceAll('//TAB//', '<b class="tab">Tab</b>');
+    report.outerHTML = reportHtml;
```

Comment:
I understand tht you duplicated the logic of zws into tab but:
* Why have you removed all the rest of the code ?
* Previously this was controlled by an option. This isn't anymore. Why ?